pb-rs/src/lib.rs

```diff
@@ -10,3 +10,109 @@ pub mod keywords;
 pub mod parser;
 pub mod scc;
 pub mod types;
+
+use errors::Error;
+use types::{Config, FileDescriptor};
+use std::path::PathBuf;
+use failure::ResultExt;
+
+#[derive(Debug, Default)]
+pub struct CompileBuilder {
```

Comment:
I would prefer having a `ConfigBuilder` only. There is nothing *compiled* in my opinion.

Reply:
Sure. I'll rename it to `ConfigBuilder`.